lib/krampus_tasks.py

```diff
@@ -239,35 +286,46 @@ def getTasks(self, key):
         try:  # we'll actually want to save this for later to rebuild task list
             self.json_data = json.load(self.bucket.Object(key).get()['Body'])
         except ClientError as e:
-            KLog.log("failed to download tasks file: %s" % str(e), "critical")
+            print "[!] failed to download tasks file: {0}".format(str(e))
+            KLog.log("Failed to download tasks file: {0}".format(str(e)), "critical")
             exit()
+
         for job in self.json_data['tasks']:
             # resolve the arn
             arn_obj = KTask.ARN(job[KEYS['arn']])
-            obj_type = arn_obj.service
-            # first, is this something we should worry about right now?
+            obj_service = arn_obj.service
+            obj_account_id = arn_obj.account_id
+
+            # Skip task if AWS Managed
+            if obj_account_id == 'aws':
```

Comment:
I would make this more explicit for IAM policies that are in the tasks list. There may be other resources with an "aws" account id that can be disabled/killed in the future.

- have `arn:aws:iam::aws:policy/ReadOnlyAccess` hit this exclusion
- have `arn:aws:iam::012345678912:policy/created-policy` not hit this exclusion
- have `arn:aws:iam::aws:fake/Thing` not hit this exclusion